# axis ids
joypad_axis_steering: 0
joypad_axis_head_pan: 5
joypad_axis_head_tilt: 6
joypad_axis_head_pan_2: -1
joypad_axis_head_tilt_2: -1

# button ids
joypad_button_forward: 1
joypad_button_all_hold: 5
joypad_button_steering_sensitivity_plus: -1
joypad_button_steering_sensitivity_minus: -1
joypad_button_head_sensitivity_plus: -1
joypad_button_head_sensitivity_minus: -1
joypad_button_head_to_default: 3
joypad_button_steering_to_default: 4

# correction factors for axis (negative values: invert direction)
steering_sensitivity: -0.08
head_pan_sensitivity: 0.05
head_tilt_sensitivity: -0.05
